.github/workflows/ci.yml

```
@@ -17,11 +17,29 @@ jobs:
       - name: Derive appropriate SHAs for base and head for `nx affected` commands
         uses: nrwl/nx-set-shas@v2
 
-      - run: npm ci
+      - name: Cache dependencies
+        uses: actions/cache@v2
+        with:
+          path: ~/.npm
+          key: npm-${{ hashFiles('package-lock.json') }}
+          restore-keys: npm-
+
+      - name: Install dependencies
+        run: npm ci --ignore-scripts
+
       - run: npx nx workspace-lint
       - run: npx nx format:check --base=origin/main
       - run: npx nx clear-cache
       - run: npx nx affected --target=test --base=origin/main --parallel --max-parallel=3
       - run: npx nx affected --target=lint --base=origin/main --parallel --max-parallel=3
       - run: npx nx affected --target=build --base=origin/main --prod --parallel --max-parallel=3
       - run: npx nx affected --target=e2e --base=origin/main --prod --parallel --max-parallel=3
+      - name: 'Build doc app for firebase hosting'
+        run: npx nx build doc
+      - name: 'Upload to Firebase HOSTING'
+        uses: FirebaseExtended/action-hosting-deploy@v0
+        with:
+          repoToken: '${{ secrets.GITHUB_TOKEN }}'
+          firebaseServiceAccount: '${{ secrets.FIREBASE_SERVICE_ACCOUNT_PRIZMUI }}'
+          expires: 30d

```

Давайка все таки уменьшим лимиты, не так много места там
```suggestion
          expires: 5d
```

---

Done